### ISSUE TITLE
Upsetplot is cropped

### ISSUE THREAD

**Issue by nina-hahn** [YuLab-SMU/clusterProfiler#696]
Dear all, 

I used the upsetplot for the kegg results. It is cropped so that I do not see any of the term names. It seems to be too big. How can I change the size? 

![image](https://github.com/YuLab-SMU/clusterProfiler/assets/109204584/ea59d05a-092c-49d7-acd0-5fce6d198c69)

Best,
Nina

**Comment by zoqaiyum** [comment]
@nina-hahn , I am facing the same issue in my upsetplot :( I am also unable to add numbers on top of the bars.

**Comment by nina-hahn** [comment]
@zoqaiyum 
I did not find a solution but managed to improve it using:

```
x<-upsetplot(gse.BP)
x+theme(plot.margin = margin(t=0.5,
                             r=0.5,
                             b=0.5,
                             l=6,
                             unit = "cm"))
```

**Comment by zoqaiyum** [comment]
@nina-hahn , this worked for me too! Thank you very much. Would you have any idea on how to add numbers on top of the bars and maybe see which genes are common between the pathways?

**Comment by nina-hahn** [comment]
No, numbers would be great but I do not have a suggestion for that.
